miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -745,7 +745,16 @@ HotTarget.sample = (function() {
             }
           },
           HotUtils.printAction('sample'),
-          HotUtils.spreadsheetAction('/miso/rest/sample/spreadsheet', Constants.sampleSpreadsheets),
+          HotUtils.spreadsheetAction('/miso/rest/sample/spreadsheet', Constants.sampleSpreadsheets, function(samples, spreadsheet){
+            var errors = [];
+            var classes = getSampleClasses(samples);
+            if(!classes.every(function(clazz){
+              return spreadsheet.sheet.allowedClasses.includes(clazz.sampleCategory);
+            })){
+              errors.push("Error: Invalid sample type");
```

Comment:
Can this error message be updated to give the user more specific information, or if there's anything they can do to fix it? ("Sample Type" is a different sample attribute, so it shouldn't be used in this error text)